libmcount/internal.h

```diff
@@ -415,6 +415,12 @@ struct mcount_disasm_engine {
 #define INSTRUMENT_FAILED                       -1
 #define INSTRUMENT_SKIPPED                      -2
 
+/* Supposing the size of smallest conditional branch is 1 byte.
+ * We can replace, at most, 5 of them by the instrumentation 
+ * instruction.
+ */
+#define MAX_COND_BRANCH 5
```

Comment:
No, I think 3 is enough for now.  The prologue size needed for other arch is different and they usually require only 2 instructions.

Reply:
I was taking about safety for other arch. I'll change it back to 3.